### ISSUE TITLE
VSMYSQL CREATE ADAPTER SCRIPT in User Guide

### ISSUE THREAD

**Issue by allipatev** [exasol/mysql-virtual-schema#36]
Customer had a problem because of `--/` before the line
```sql
CREATE OR REPLACE JAVA ADAPTER SCRIPT SCHEMA_FOR_VS_SCRIPT.ADAPTER_SCRIPT_MYSQL AS
```
as SQL clients other then DB Visualizer might find the script body to send to DB incorrectly in this case.

Please consider adding a remark to the User guide that the comment ('--/') will is ONLY necessary for a DB Visualizer SQL client and not for others.

**Comment by ckunki** [comment]
Already solved with #26

**Comment by ckunki** [comment]
See https://github.com/exasol/mysql-virtual-schema/blob/main/doc/user_guide/mysql_user_guide.md#installing-the-adapter-script

**Comment by ckunki** [comment]
Hello @allipatev sorry I misread the current ticket.

**Comment by ckunki** [comment]
Hello @allipatev I also checked the official documentation at [docs.exasol.com](https://docs.exasol.com/db/latest/database_concepts/virtual_schema/user_guide.htm) and noticed 
* it contains the prefix `--/` without delimitation 
* but adds a reference to [SQL Client Troubleshooting](https://docs.exasol.com/db/latest/database_concepts/virtual_schema/user_guide.htm#SQL_Client_Troubleshooting) section. 

The first line is a comment and should be ignored by SQL clients AFAIK.
I also checked the script with SQL Client [DBeaver](https://dbeaver.io/) without any problems which means for DBeaver the first line is not required but also not harmful.

If possible I also would prefer to keep the documentation as simple and straight-forward as possible, i.e. avoid if-then cases describing different scenarios whenever possible. 

Would it be acceptable in your eyes if VSMYSQL user guide would use the same strategy as the official documentation?
